ionic-app/www/js/directives/sessionDirective.js

```diff
@@ -0,0 +1,9 @@
+app_directives.directive('dddSession', function () {
```

Comment:
This message came to my email box. :-). Just attach the screenshots of pull
#7 for a ref.

On Mon, May 2, 2016 at 1:52 AM, Claudio Beatrice <notifications@github.com>
wrote:

> In ionic-app/www/js/directives/sessionDirective.js
> <https://github.com/drupaldevdays/mobile/pull/10#discussion_r61694611>:
>
> > @@ -0,0 +1,9 @@
> > +app_directives.directive('dddSession', function () {
>
> I find the naming of this directive and the template it's using a bit
> misleading
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/drupaldevdays/mobile/pull/10/files/85191ab0edd709ced60dda53c268fdb8016e24d8#r61694611>
>